gravatar-ui/build.gradle.kts

```diff
@@ -1,19 +1,16 @@
 import org.jetbrains.dokka.gradle.DokkaTaskPartial
 
 plugins {
-    id("com.android.library")
-    id("org.jetbrains.kotlin.android")
-    id("org.jetbrains.kotlin.plugin.compose")
-    id("com.automattic.android.publish-to-s3")
-
+    id(libs.plugins.android.library.get().pluginId)
```

Comment:
I could not use the `alias()` here, the Gradle sync would error out with [this issue](https://github.com/gradle/gradle/issues/20084).

Reply:
TIL

Reply:
I reckon you could have used `alias` here if you'd added

```
alias(libs.plugins.android.library) apply false
```

to the root `build.gradle.kts` too, just like it's done for `alias(libs.plugins.android.application) apply false`.

Reply:
You are right. I've opened a ticket (#432) and PR (#433), so feel free to leave any comments there.

Thanks for your suggestions, @davidburstrom; we are delighted to receive external feedback and contributions!

Reply:
My pleasure! I happened to come across it as @AdamGrzybkowski linked to the Gradle issue, and I also had a solution for it.